magenta-lib/src/main/scala/magenta/graph/Graph.scala

```diff
@@ -113,13 +180,17 @@ case class Graph[T](edges: Set[Edge[T]]) {
   lazy val toList: List[T] = nodeList.flatMap(_.maybeValue)
 
   override def toString: String = {
```

Comment:
Would an output like the `tree` command's be easier to visualise than separate nodes and edges?

If I remember (and see) correctly, 'priority' really means order, so it's captured by an ordered representation without needing to be displayed separately?